### ISSUE TITLE
Org-agenda files as org-brain files

### ISSUE THREAD

**Issue by JacintoCC** [Kungsgeten/org-brain#247]
I have large agenda files that are considered as org-brain files since the last update. Is there any way of stopping this behaviour, as it slows all the processes?

**Comment by Kungsgeten** [comment]
Hmm, that seems strange. Have you tried running `M-x org-brain-update-id-locations`?

**Comment by JacintoCC** [comment]
Yes. In fact, if I run `org-brain-update-id-locations` before opening any org-brain files, this function only finds entries in these agenda files.
